elvish-mode.el

```diff
@@ -93,8 +104,80 @@ expected.")
       (optional "." (optional (one-or-more digit))))
   "The regex to identify elvish numbers")
 
+(defcustom elvish-builtin-functions
+  '(
+    ;; Trivial builtin
+    "nop"
+    ;; Introspection
+    "kind-of" "is" "eq"
+    ;; Value output
+    "put"
+    ;; Bytes output
+    "print" "echo" "pprint" "repr"
+    ;; Bytes to value
+    "slurp" "from-lines" "from-json"
+    ;; Value to bytes
+    "to-lines" "to-json"
+    ;; Exception and control
+    "fail" "multi-error" "return" "break" "continue"
+    ;; Misc functional
+    "constantly"
+    ;; Misc shell basic
+    "-source"
+    ;; Iterations.
+    "each" "peach" "repeat"
+    ;; Container primitives.
+    "assoc"
+    ;; Sequence primitives
+    "explode" "take" "range" "count" "has-key" "has-value"
+    ;; String
+    "joins" "splits"
+    ;; String operations
+    "ord" "base" "wcswidth" "-override-wcwidth"
+    ;; Map operations
+    "keys"
+    ;; String predicates
+    "has-prefix" "has-suffix"
+    ;; String comparison
+    "<s" "<=s" "==s" "!=s" ">s" ">=s"
+    ;; eawk
+    "eawk"
+    ;; Directory
+    "cd" "dir-history"
+    ;; Path
+    "path-abs" "path-base" "path-clean" "path-dir" "path-ext" "eval-symlinks" "tilde-abbr"
+    ;; Boolean operations
+    "bool" "not"
+    ;; Arithmetics
+    "+" "-" "*" "/" "^" "%"
```

Comment:
We should probably drop these for now. Otherwise, we get things like `find /` highlighting the '/'. The same thing happens with the comparisons and redirects (`ls > test` highlights the '>').